conf/deployment/fusion_hci_pc/provider_bm_upi_1az_rhcos_nvme_3m_3w.yaml

```diff
@@ -0,0 +1,13 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+  local_storage: true
+ENV_DATA:
+  platform: 'baremetal'
+  cluster_type: 'provider'
```

Comment:
If we're going to have hci_client, why don't we introduce hci_provider to distinguish it from old provider cluster?

Reply:
To minimise the changes for exiting working `runs_on_provider`